src/st-flash/flash_opts.c

```diff
@@ -2,11 +2,12 @@
 #include <stdio.h>
 #include <string.h>
 
-#include <flash.h>
+#include "flash.h"
```

Comment:
We can leave the brackets < > here, as our makefile explicitly takes care of finding this header file.
Thus it is unified throughout the codebase.

Reply:
Correction: My mistake. It's ok like this. Just leave it as it is.